src/main/java/nonapi/io/github/classgraph/json/JSONUtils.java

```diff
@@ -76,7 +76,7 @@
             final char hexDigit1 = nibble1 <= 9 ? (char) ('0' + nibble1) : (char) ('A' + nibble1 - 10);
             final int nibble0 = c & 0xf;
             final char hexDigit0 = nibble0 <= 9 ? (char) ('0' + nibble0) : (char) ('A' + nibble0 - 10);
-            JSON_CHAR_REPLACEMENTS[c] = "\\u00" + Character.toString(hexDigit1) + Character.toString(hexDigit0);
+            JSON_CHAR_REPLACEMENTS[c] = "\\u00" + hexDigit1 + hexDigit0;
```

Comment:
This looks dangerous to me... probably `+` will be applied left-to-right, but adding two `char`s that are supposed to be string-concatenated is dodgy, because integer addition will be applied if you're not careful

Reply:
I rewrote this as `"\\u00" + hexDigit1 + "" + hexDigit0` just to be a bit safer / more explicit. This will probably be caught by a static analyzer too though as a no-op string concatenation.